src/documents/UserInterfaceOptionsAPI.md

```diff
@@ -127,8 +127,8 @@ For information on the different types of events, see [Infusion Event System](In
             <th>Description</th>
             <td>
                 This event fires when the UI Options interface has been rendered into the iframe.
-
-                <p><em><strong>Note:</strong> use <code>onReady</code> if the listener needs UI Options to be both rendered and ready to use.</em></p>
+                <br><br>
+                <em><strong>Note:</strong> use <code>onReady</code> if the listener needs UI Options to be both rendered and ready to use.</em>
```

Comment:
You shouldn't need the `<br><br>`. It seems this "NOTE" is just marked up incorrectly. It should look more like. 

```html
<div class="infusion-docs-note">
    <strong>Note:</strong> Use <code>onReady</code> if the listener needs UI Options to be both rendered and ready to use.
</div>
```